source/community.applications/usr/local/emhttp/plugins/community.applications/include/helpers.php

```
@@ -1178,7 +1178,12 @@ function makeXML($template) {
 		}
 	}
 	$Array2XML = new Array2XML();
-	$xml = $Array2XML->createXML("Container",$template);
+	try {
+		$xml = $Array2XML->createXML("Container",$template);
+	} catch (Throwable $e) {
+		debug("makeXML: Array2XML failed for ".($template['Name'] ?? 'unknown')." - ".$e->getMessage());
+		return false;
```

PHPStan (2.2.5)

[error] Function makeXML() should return string but returns false.


(return.type)